PSI-MOD.obo

```diff
@@ -41152,9 +41153,17 @@ xref: UniProt: "PTM-0691"
 is_a: MOD:00599 ! monomethylated residue
 is_a: MOD:00602 ! N-methylated residue
 is_a: MOD:00673 ! methylated asparagine
-is_a: MOD:00203 ! D-asparagine
+is_a: MOD:02097 ! modified D-asparagine residue
 is_a: MOD:00894 ! residues isobaric at 128.058578 Da
 
+[Term]
+id: MOD:02097
+name: modified D-asparagine residue
+def: "A protein modification that modifies a D-asparagine residue." [PubMed:18688235]
+subset: PSI-MOD-slim
+synonym: "ModDAsn" EXACT PSI-MOD-label []
+is_a: MOD:00203 ! D-asparagine

```

Comment:
a modified AA cannot have as parent a non modified residue, except in case of crosslinks that only adds a linker or removes a H2/NH2/H2O (a t least this is how it is implemented so far)

remove is_a: MOD:00203 ! D-asparagine
add is_a: MOD:01157 ! protein modification categorized by amino acid modified

Reply:
I will change this in the next version.  D amino acids are not implemented optimally.  I've created issue #58 to address this.